libraries/client/src/main/aidl/org/holochain/androidserviceruntime/client/IHolochainServiceCallback.aidl

```diff
@@ -14,4 +17,8 @@ interface IHolochainServiceCallback {
     void isAppInstalled(boolean response);
     void ensureAppWebsocket(in AppAuthFfiParcel response);
     void signZomeCall(in ZomeCallFfiParcel response);
+
+    // Error responses
+    void adminBinderUnauthorizedException(in AdminBinderUnauthorizedExceptionParcel response);
+    void appBinderUnauthorizedException(in AppBinderUnauthorizedExceptionParcel response);
```

Comment:
We cannot throw an exception over the client-service ipc connection between processes. So instead we add error callbacks.